docs/pages/guides/environment-variables.mdx

```diff
@@ -103,17 +103,17 @@ function Post() {
 
 This approach this plugin will work well with environment variables set in your shell process, but it won't automatically load **.env** files. We recommend using [direnv](https://direnv.net/) and **.envrc** to accomplish that.
 
-## Storing environment variables in .envrc or .env
+## Loading environment variables from a file
 
-If you want to automatically load environment variables from a file in your project rather than setting them in your shell profile or manually when running `npx expo start`, you may want to use a **.envrc** or **.env** file.
+If you want to automatically load environment variables from a file in your project rather than setting them in your shell profile or manually when running `npx expo start`, you may want to use an environment variable manager.
```

Comment:
```suggestion
If you want to automatically load environment variables from a file in your project rather than setting them in your shell profile or manually when running `npx expo start`, you may want to use an environment variable manager such as direnv or dotenv.
```

Reply:
Eh, your bold below helps, and "such as" also helps, I'm good with it :-)

Reply:
We can remove the phrase if you think the addition above doesn't add anything or is going to lead to confusion.

One way I think is, at times, we should spell out terms we want to communicate (it has small benefits, for example, in some cases a screen reader will pick this up or somebody skimming will go through first and last words of a paragraph, etc.) That was my intention behind it.